modules/ztp-coordinating-reboots-for-config-changes.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.adoc
```

Comment:
Should this be `edge_computing/policygenerator_for_ztp/ztp-configuring-managed-clusters-policygenerator.adoc` ? because you added `modules/ztp-coordinating-reboots-for-config-changes.adoc` in that assembly?

So whichever is the correct assembly, enter the assembly's context in `ifeval` condition.